src/app/components/score/scoreboard-selector/scoreboard-selector.vue

```diff
@@ -1,62 +1,14 @@
 <template>
-	<div class="scoreboard-selector">
-		<div class="list-group">
-			<!--
-			When more than one, you can click to change.
-		-->
-			<template v-if="tables.length > 1">
-				<app-popper block track-trigger-width>
-					<a class="list-group-item has-addon">
-						<div class="list-group-item-addon">
-							<app-jolticon icon="chevron-down" />
-						</div>
-
-						<h5 class="list-group-item-heading sans-margin-bottom">
-							<strong>{{ currentTable.name }}</strong>
-						</h5>
-						<p class="list-group-item-text">
-							{{ currentTable.description }}
-						</p>
-					</a>
-
-					<div slot="popover">
-						<div class="popper-heading">
-							<strong>
-								<translate>scores.scoreboard_popover.heading</translate>
-							</strong>
-						</div>
-
-						<div class="list-group">
-							<a
-								class="list-group-item has-icon"
-								v-for="table of tables"
-								:key="table.id"
-								@click="selectTable(table)"
-							>
-								<h5 class="list-group-item-heading sans-margin-bottom">
-									<app-jolticon icon="check" v-if="table.id === currentTable.id" />
-									<strong>{{ table.name }}</strong>
-								</h5>
-
-								<div class="list-group-item-text">{{ table.description }}</div>
-							</a>
-						</div>
-					</div>
-				</app-popper>
-			</template>
-			<!--
-			When only one, it's not clickable.
-		-->
-			<div class="list-group-item" v-else>
-				<h5 class="list-group-item-heading sans-margin-bottom">
-					<strong>{{ currentTable.name }}</strong>
-				</h5>
-				<p class="list-group-item-text">
-					{{ currentTable.description }}
-				</p>
-			</div>
-		</div>
-	</div>
+	<app-list-group-selector :items="tables" :current="currentTable" @change="selectTable($event)">
```

Comment:
Doesn't seem to scroll with the content.
![image](https://user-images.githubusercontent.com/996947/68131906-7fd4bf80-ff26-11e9-9867-1ed924d4aa2c.png)

Reply:
Current issue, not something this PR introduced.